src/core/markdown/MarkupManager.ts

```diff
@@ -0,0 +1,89 @@
+import {EventEmitter} from 'events';
```

Comment:
We have our own implementation of event emitter: https://github.com/gravity-ui/markdown-editor/blob/main/src/utils/event-emitter.ts

Reply:
fixed